archesproject/arches-for-science-prj
# Manifest manager loads in workflows

Solves #290 

To test:
1. Start Cantaloupe
2. Navigate to the Access Areas workflow
3. In the `Image` tab, press the `Create new Image Service` button
4. Notice how the manifest manager loads and is operable
